src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -629,6 +629,20 @@ Your only job is to make sure that the CSRF component is installed:
 
     $ composer require symfony/security-csrf
 
+If you want to disable CSRF for a single component you can set
+`csrf` option to `false`::
```

Comment:
FYI: it's RST, so we need TWO "ticks" on either side. But I'll fix this real quick after merge :)

Reply:
fixed :)